osu.Framework.Android/Input/AndroidTouchHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
+
+using Android.Views;
+using osu.Framework.Input.Handlers;
+using osu.Framework.Input.StateChanges;
+using osu.Framework.Platform;
+using osuTK.Input;
+
+namespace osu.Framework.Android.Input
+{
+    public class AndroidTouchHandler : InputHandler
+    {
+        private readonly AndroidGameView view;
+
+        private float LastX, LastY;
+
+        public AndroidTouchHandler(AndroidGameView view)
+        {
+            this.view = view;
+            view.Touch += HandleTouches;
+        }
+
+        private void HandleTouches(object sender, View.TouchEventArgs e)
```

Comment:
Make private methods lowercase
```suggestion
        private void handleTouches(object sender, View.TouchEventArgs e)
```